cvmfs/mountpoint.h

```diff
@@ -219,7 +220,7 @@ class FileSystem : SingleCopy, public BootFactory {
   bool LockWorkspace();
   bool SetupCrashGuard();
   bool CreateCache();
-  bool SetupQuotaMgmt();
+  bool SetupQuotaMgmt(cache::CacheManagerIds mgr_type);
```

Comment:
I think we'd want to store the cache manager type as a class member variable of `MountPoint`.